app/assets/stylesheets/hoge/fuga.css.scss

```diff
@@ -0,0 +1,40 @@
+.cat{}
+
+.dog {
+  color: #ff00ff;
```

Comment:
__[scss_lint]__
* [ColorVariable] Color literals like `#ff00ff` should only be used in variable declarations; they should be referred to via variable everywhere else.
* [HexLength] Color `#ff00ff` should be written as `#f0f`

<!-- Comment by [SideCI](https://sideci.com) -->